presto-main/src/main/java/com/facebook/presto/operator/LookupJoinOperator.java

```diff
@@ -16,31 +16,46 @@
 import com.facebook.presto.operator.LookupJoinOperators.JoinType;
 import com.facebook.presto.spi.Page;
 import com.facebook.presto.spi.PageBuilder;
+import com.facebook.presto.spi.block.Block;
```

Comment:
please review rewritten commit (and refactor commit which is at begin of the PR)